Reaqtor/Core/Expressions/Reaqtor.Expressions.Core/Reaqtor/ReactiveQbservable.cs

```diff
@@ -502,6 +502,60 @@ public static IReactiveQbservable<TSource> Finally<TSource>(this IReactiveQbserv
 
         #region Reactive Operators
 
+        #region Distinct
+
+        /// <summary>
+        /// Propagates distinct elements from the source sequence using the default comparer for elements.
+        /// </summary>
+        /// <typeparam name="TSource">Type of the elements in the source sequence.</typeparam>
+        /// <param name="source">Source sequence whose elements to propagate.</param>
+        /// <returns>Observable sequence propagating the source sequence's distinct elements.</returns>
+        public static IReactiveQbservable<TSource> Distinct<TSource>(this IReactiveQbservable<TSource> source)
+        {
+            throw new NotImplementedException();
```

Comment:
I'd welcome a separate PR (to avoid unrelated stylistics changes here) that changes all of these to use the `=>` expression bodied syntax for all of these `throw` statements.

Reply:
Sure